README.md

```
@@ -47,38 +47,189 @@ parser parse -f markdown https://example.com/article
+        log.Fatal(err)
+    }
+    
+    fmt.Printf("Title: %s\n", result.Title)
+    fmt.Printf("Content: %s\n", result.Content)
+}
+```
+
+## Migration from v0.x to v1.0
+
+If you're upgrading from the old internal API, here are the key changes:
+
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)